aws-ssm-patchbaseline/.gitignore

```diff
@@ -12,9 +12,15 @@ out.java
 out/
 .settings
 .project
-
+*.whl
 # auto-generated files
 target/
+*_internal.json
 
 # our logs
 rpdk.log
+/.apt_generated/
```

Comment:
what generated those?

Reply:
Now it is removed and removed the entry in the gitignore file